gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherApi.java

```diff
@@ -321,6 +309,7 @@ public PublishResponse publish(PublishRequest request) {
    * <!-- end manual edit -->
    *
    * @param topic The name of the topic to get.
```

Comment:
Nit, code generation comment: param description should start with lower case (like the description for throws, return,...) and does not need a period). For more see [this](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#tag).